src/duplicacy_gcdstorage.go

```diff
@@ -30,8 +30,8 @@ type GCDStorage struct {
 	service         *drive.Service
 	idCache         map[string]string
 	idCacheLock     *sync.Mutex
-	backoffs        []float64
-	backoffsRetries []int
+	backoffs        []float64  // desired backoff time in seconds for each thread
+	attempts        []int      // number of tries for each thread
```

Comment:
This comment doesn't explain what this is very clearly. Maybe "number of failed attempts since the last success for each thread"?